crates/wiggle/src/lib.rs

```diff
@@ -953,3 +953,40 @@ impl From<GuestError> for Trap {
         Trap::String(err.to_string())
     }
 }
+
+pub unsafe fn run_in_dummy_executor<F: std::future::Future>(future: F) -> F::Output {
```

Comment:
I think this technically isn't `unsafe`, right?

Reply:
Probably not - there is some `unsafe` under the hood but afaict that shouldnt make it unsafe itself. I debated about that keyword and ultimately threw it in there to flag "don't use this unless you are really sure", initially this was totally private code but I needed to reuse it in several crates.